libraries/stdlib/samples/test/samples/collections/iterables.kt

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2010-2017 JetBrains s.r.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package samples.collections
+
+import samples.*
+import kotlin.test.*
+
+class Iterables {
+
+    @Sample
+    fun iterable() {
+        val iterator = Sequence { arrayOf(1, 2, 3).iterator() }.iterator()
```

Comment:
This sample doesn't use the function it is attached to.
The function may be useful for turning into `Iterable` something that has iterator or is an `Iterator`, but isn't `Iterable` itself, for example `buildIterator { }`